.github/workflows/cicd.yml

```
@@ -392,3 +392,17 @@ jobs:
       code_coverage: false
       gather_dumps: true
       fault_injection: true
+
+  # Run multi-threaded stress tests against the user mode 'mock' framework.
+  user_mode_multi_threaded_stress_test:
+    needs: regular
+    uses: ./.github/workflows/reusable-test.yml
```

```suggestion
    if: github.event_name == 'schedule'
    uses: ./.github/workflows/reusable-test.yml
```
Don't you need the 'if' like line 385 has?

---

Looks like it ran on this pull request (due to the lack of the `if` noted above) and the run failed.  See
https://github.com/microsoft/ebpf-for-windows/actions/runs/4474989790/jobs/7864182352?pr=2195

---

fixed.